submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionFilesMongoPersistenceService.kt

```diff
@@ -28,26 +30,13 @@ class SubmissionFilesMongoPersistenceService(
         submissionFilesRepository.save(docFile)
     }
 
-    override fun getSubmissionFile(path: String, accNo: String, version: Int): ExtFile {
-        val docFile = submissionFilesRepository.getByPathAndAccNoAndVersion(path, accNo, version)
-        return docFile.file.toExtFile()
-    }
-
-    override fun getSubmissionFiles(accNo: String, version: Int, startingAt: Int): List<Pair<ExtFile, Int>> {
-        return submissionFilesRepository
-            .findAllByAccNoAndVersionAndIndexGreaterThan(accNo, version, startingAt)
-            .map { (it.file.toExtFile() to it.index) }
-    }
-
-    override fun getFileListFiles(accNo: String, version: Int, fileListName: String): List<ExtFile> {
-        return submissionFilesRepository
-            .findAllByAccNoAndVersionAndFileList(accNo, version, fileListName)
-            .map { it.file.toExtFile() }
-    }
-
     override fun getReferencedFiles(accNo: String, fileListName: String): List<ExtFile> {
```

Comment:
I'm moving this operation back to the `SubmissionPersistenceQueryService` API since we agreed that the files in the request collection will be different from the actual submission files once we decouple them.

Could you please do the same so I don't get into any conflicts later during merging? 😅